src/app/shared/subscriptions/subscription-view/subscription-view.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Subscription } from '../models/subscription.model';
+import { DSpaceObject } from '../../../core/shared/dspace-object.model';
+
+import { take } from 'rxjs/operators';
+import { NgbModal, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+
+import { hasValue } from '../../empty.util';
+import { ConfirmationModalComponent } from '../../confirmation-modal/confirmation-modal.component';
+import { SubscriptionsDataService } from '../subscriptions-data.service';
+import { getCommunityModuleRoute } from '../../../community-page/community-page-routing-paths';
+import { getCollectionModuleRoute } from '../../../collection-page/collection-page-routing-paths';
+import { getItemModuleRoute } from '../../../item-page/item-page-routing-paths';
+import { SubscriptionModalComponent } from '../subscription-modal/subscription-modal.component';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
+  selector: '[ds-subscription-view]',
+  templateUrl: './subscription-view.component.html',
+  styleUrls: ['./subscription-view.component.scss']
+})
+export class SubscriptionViewComponent {
```

Comment:
Again please add a description or TypeDocs to describe how/where this component is used.